rebasehelper/cli.py

```diff
@@ -143,6 +150,13 @@ def add_args(self):
             help="Specify number of retries in case build fails.",
             type=int
         )
+        self.parser.add_argument(
+            "--enable-option",
```

Comment:
@Jurisak So my initial ideas were ```--builder-params``` or ```--builder-args```. IMHO there is no need to use "enable" in the option name.